src/app/(BottomTabNavigation)/AllCases/Updates/UpdateView/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { useLocalSearchParams } from 'expo-router';
+import React, { useEffect, useState } from 'react';
+import { View, Text, ScrollView } from 'react-native';
+
+import styles from './styles';
+import { getUpdateById } from '../../../../../supabase/queries/updates';
+import { Update, UpdateUid } from '../../../../../types/types';
+import { formatDate } from '../../utils';
+
+export default function UpdateView() {
+  const { updateUid, lawFirm } = useLocalSearchParams<{
+    updateUid: UpdateUid;
+    lawFirm: string;
+  }>();
```

Comment:
Can we update this file to take the `updateUid` as the search parameter through the name of the file? We should also add the `lawFirm` to the update row in the supabase table. This should make integrating notifications and deeplinking easier.